mysql-test/main/outfile_fifo.test

```diff
@@ -0,0 +1,17 @@
+#
+# MDEV-32061: Allow INTO OUTFILE to writo to a fifo
```

Comment:
typo + convention is to `--echo # MDEV....`

Reply:
Done.